src/codegen.cpp

```diff
@@ -614,7 +614,7 @@ static const auto jl_newbits_func = new JuliaFunction{
                 {T_prjlvalue, T_pint8}, false); },
     [](LLVMContext &C) { return AttributeList::get(C,
             AttributeSet(),
-            Attributes(C, {Attribute::NoAlias, Attribute::NonNull}),
+            Attributes(C, {Attribute::NonNull}),
```

Comment:
backport?

Reply:
Oh yeah, this line of change is in principle a bug fix..... (commit was from a week ago and I've forgot about that...............)

Reply:
Yeah, I think I missed the second half of the langref documentation for this attribute. (for arguments, it means that it doesn't alias an mutable memory, but for return values, it also means the pointer itself is unique à la malloc)